pkg/containerinstall/manager.go

```diff
@@ -0,0 +1,46 @@
+package containerinstall
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+
+	"github.com/sirupsen/logrus"
+
+	"github.com/Azure/ARO-RP/pkg/api"
+	"github.com/Azure/ARO-RP/pkg/env"
+)
+
+type ContainerInstaller interface {
+	Install(ctx context.Context, sub *api.SubscriptionDocument, doc *api.OpenShiftClusterDocument, version *api.OpenShiftVersion) error
+}
```

Comment:
Could we maybe add a setup function or something? Because getting the env variables directly inside the install function makes it tightly bound with the environment

Reply:
I tweaked this a bit more, but since it all comes from the dev environment it's a bit of a hassle to do super nicely unless we were going to use it in not-dev